static/.well-known/security.txt

```diff
@@ -1,7 +1,10 @@
-Contact: mailto:security@scratchaddons.com
-Preferred-Languages: en
 Policy: https://github.com/ScratchAddons/ScratchAddons/security/policy
 Policy: https://scratchaddons.com/docs/policies/security/
+
+Contact: https://github.com/ScratchAddons/ScratchAddons/security/advisories/new
+Contact: mailto:security@scratchaddons.com
+Preferred-Languages: en
+
 Acknowledgements: https://scratchaddons.com/credits
 
-Expires: 2023-01-01T00:00:00.000Z

```

Comment:
The build script will always set the date to 4 weeks after the build date. [What is live on the website](https://scratchaddons.com/.well-known/security.txt) is set to 4 January 2024.

When I was adding it, my plan is to update the static source every year, so in 2023 I would update it to 2024. Obviously I forgot about it, so instead of doing it for 2024, I decided to do it to 2025 instead.